Dockerfile

```diff
@@ -31,11 +31,11 @@ RUN npm run build
 FROM ipfs/kubo:v0.25.0@sha256:0c17b91cab8ada485f253e204236b712d0965f3d463cb5b60639ddd2291e7c52 as ipfs-kubo
 
 # Create the base image
-FROM debian:12.2-slim@sha256:93ff361288a7c365614a5791efa3633ce4224542afb6b53a1790330a8e52fc7d
```

Comment:
Is this necessary to fix the problem, or can we *just* update the curl version?

Reply:
the linux version we are using has bunch of security issues identified. I think it makes sense to update it